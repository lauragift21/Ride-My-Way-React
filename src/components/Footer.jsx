import React from "react";

const Footer = () => (
  <footer className="copyright">
    <div className="container">
      <h4> Â© 2018 RideMyWay Inc.</h4>
    </div>
  </footer>
);
export default Footer;
