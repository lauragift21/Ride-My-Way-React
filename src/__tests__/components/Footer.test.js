import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Footer from "../../components/Footer";

describe("Footer Component", () => {
  test("renders the Footer Component", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.exists()).toBe(true);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
