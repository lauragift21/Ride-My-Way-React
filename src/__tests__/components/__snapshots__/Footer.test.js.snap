// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders the Footer Component 1`] = `
<footer
  className="copyright"
>
  <div
    className="container"
  >
    <h4>
       Â© 2018 RideMyWay Inc.
    </h4>
  </div>
</footer>
`;
